Dockerfile

```diff
@@ -1,8 +1,172 @@
-FROM hathitrust/feed_base:buster
+# FROM hathitrust/feed_base:bullseye
 
-RUN apt-get install -y libtest-class-perl libswitch-perl libtest-spec-perl libtest-mockobject-perl
+FROM debian:bullseye
 
-RUN wget -O /tmp/netpbm-sf-10.73.32_amd64.deb https://sourceforge.net/projects/netpbm/files/super_stable/10.73.32/netpbm-sf-10.73.32_amd64.deb/download
+RUN sed -i 's/main.*/main contrib non-free/' /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y \
```

Comment:
Do we know for sure all these are required, or is this just what was in the old `feed_base` image?

Reply:
It's a bit of old `feed_base` and what is in the CRMS `Dockerfile`.